src/main/xar-resources/modules/app.xql

```diff
@@ -108,7 +108,7 @@ function app:module($node as node(), $model as map(*), $details as xs:boolean) {
     
     order by $order
     return
-        app:print-module($module, $funcsInModule, $details)
+        app:print-module($module, $funcsInModule, boolean($details))
```

Comment:
Is this change necessary? The function signature already defines `$details as xs:boolean`.

Reply:
I had to make that change, I believe. Have to check the code.

Reply:
This change was actually a symptom of a bug in the templating module. I've submitted a PR fixing this: https://github.com/eXist-db/templating/pull/24. I'll merge this now and submit a follow-on PR here once the templating module fix is merged and released.